app/grandchallenge/reader_studies/models.py

```diff
@@ -1528,6 +1591,18 @@ def validate(  # noqa: C901
                 offset=question.answer_min_value,
             )(answer)
 
+        if question.answer_min_length is not None:
+            MinLengthValidator(question.answer_min_length)(answer)
+
+        if question.answer_max_length is not None:
+            MaxLengthValidator(question.answer_max_length)(answer)
+
+        if question.answer_match_pattern is not None:
```

Comment:
Should this be `is not None`? It's a non-nullable field. I think `if question.answer_match_pattern` would do.

Reply:
Good point! I had thought to replace this in another check but forgot to also do this here.